python/lsst/dax/distribution/DataGenClient.py

```diff
@@ -208,6 +217,7 @@ def _readDatagenConfig(self):
         self._edge_width = spec_globals['edge_width']
         print("_cfgFileContents=", self._cfg_file_contents)
         print("_spec=", self._spec)
+        self._data_gen = DataGenerator(self._spec)
```

Comment:
Would like to see a comment on what and why of exec() statement above.

Reply:
I added some brief comments. 'spec' itself is very complicated in what can be done with it. I expect Colin will be making significant changes to how it works and he will, hopefully, write something on how it should be used. See configs/fakedb/fakeGenSpec.py for an example.